### DISCUSSION TITLE
"There are no repositories defined in your helmfile.yaml"

### DISCUSSION THREAD

**Discussion by WoodyWoodsta** [helmfile/helmfile#321] [Questions]
I've got a directory with many helm charts, which are individual releases. I've decided to have a helmfile per release, as the releases are synonymous with a service/app. At the root, I have a single helmfile which contains any helm defaults and references to the helmfiles.

When I run `sync` or `deps`, I get loads of this message:

```
There are no repositories defined in your helmfile.yaml.
This means helmfile cannot update your dependencies or create a lock file.
See https://github.com/roboll/helmfile/issues/878 for more information.
```

But this is simply not true - not all charts require remote dependencies, and some don't require any dependencies at all.

Am I using helmfile in the wrong way or is this something that can be improved in the helmfile messaging?

**Comment by yxxhero** [comment]
@WoodyWoodsta Please put a example. So I can debug. Thanks very much.

**Reply by WoodyWoodsta** [reply]
You can use any helmfile which does not have a `repositories` field.

**Comment by yxxhero** [comment]
https://github.com/roboll/helmfile/issues/878

**Reply by WoodyWoodsta** [reply]
I've read through that thread and it does not really satisfy the problem. If I have many helmfiles without the `repositories` field, I get many duplicated warnings (sometimes 3-4 in a row, multiplied by 10s of helmfiles, that's many warning messages).
